syftlib/src/main/java/org/openmined/syft/execution/SyftJob.kt

```diff
@@ -212,12 +226,20 @@ class SyftJob internal constructor(
                             ReportRequest(
                                 workerId,
                                 requestKey,
-                                diff.serialize().toString()
+                                Base64.encodeToString(
+                                    diff.serialize().toByteArray(),
+                                    Base64.DEFAULT
+                                )
                             )
                         )
                         .compose(config.networkingSchedulers.applySingleSchedulers())
                         .subscribe { reportResponse: ReportResponse ->
-                            Log.i(TAG, reportResponse.status)
+                            if (reportResponse.error != null)
```

Comment:
I think this can be done by folding `reportResponse`. We can see what would it needs to make it. Just a comment.